src/cript/api/api.py

```diff
@@ -1241,11 +1233,13 @@ def delete_node_by_uuid(self, node_type: str, node_uuid: str) -> None:
         -------
         None
         """
-        delete_node_api_url: str = f"{self._host}/{node_type.lower()}/{node_uuid}/"
+        node_type_snake_case: str = get_node_type_snake_case(node_type=node_type)
+
+        delete_node_api_url: str = f"{self._host}/{node_type_snake_case}/{node_uuid}/"
 
         response: Dict = requests.delete(headers=self._http_headers, url=delete_node_api_url, timeout=_API_TIMEOUT).json()
 
         if response["code"] != 200:
             raise APIError(api_error=str(response), http_method="DELETE", api_url=delete_node_api_url)
 
-        self.logger.info(f"Deleted '{node_type.title()}' with UUID of '{node_uuid}' from CRIPT API.")
+        self.logger.info(f"Deleted '{node_type_snake_case.title()}' with UUID of '{node_uuid}' from CRIPT API.")
```

Comment:
use the basenode functions

Reply:
that is if I have access to the class, but if I have access to only the string or class then I'll have to write the if logic at the beginning of the file and always get the str name from the class/object